website/pages/blog/product-updates-1.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "CloudQuery Product Updates #1"
+tag: product
+date: 2022/11/01
+description: >-
+  Monthly updates on CloudQuery product and roadmap.
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Since CloudQuery [V1 is out](./cloudquery-v1-release.md) we will be restarting our monthly products updates and starting from #1 as figured it easier to use number then months :)
```

Comment:
```suggestion
Hey everyone! Since CloudQuery [V1 is out](./cloudquery-v1-release.md) we will be restarting our monthly products updates and starting from #1 as I figured it is easier to use numbers than months :)
```